examples/distributed-multigrid-preconditioned-solver/doc/intro.dox

```diff
@@ -0,0 +1,9 @@
+<a name="Intro"></a>
+<h1>Introduction</h1>
+This distributed multigrid preconditioned solver example should help you understand customizing Ginkgo multigrid in a distributed setting.
+The example will solve a simple 1D Laplace equation where the system can be distributed row-wise to multiple processes.
+Note. Because the stencil for the discretized Laplacian is configured equal weighted, the coarsening method does not perform well on this kind of problem.
```

Comment:
nit:
```suggestion
Note. Because the stencil for the discretized Laplacian is configured with equal weights, the coarsening method does not perform well on this kind of problem.
```